csp-guru/csp-guru.csv

```diff
@@ -1,110 +1,112 @@
+Lebrija,2011,operating,Spain,Lebrija 1,37.003,-6.047777778,2065,2008,Trough,50,0,412020,Wet,120,0.273972603,303,421.989615,463.777214,9275.54428,303,453.946699,9078.933981,Solucia,Soleval Renovables,Solel,Solel,FIT,0.27,0.37603035,0.413266824,0.27,0.290446979,0.332189927,0.483750832,0.325148619,0.473496943,#N/A,Solel 50%; Valoriza 50%,25,Lillestam et al 2017: Nature energy
+Gemasolar,2011,operating,Spain,Fuentes de Andalucia,37.56248611,-5.330386111,2072,2009,Tower,20,15,304750,Wet,110,0.627853881,230,320.32215,352.0421096,17602.10548,230,344.5800026,17229.00013,Torresol,UTE Solar Tres,Sener,Sener,FIT,0.27,0.37603035,0.413266824,0.27,0.290446979,0.275080767,0.400585747,0.269249981,0.392094679,EIB €110 million; ICO €16.14 million,Masdar; Sener,25,Lillestam et al 2017: Nature energy
+Arcosol 50 (Valle I),2011,operating,Spain,San José del Valle,36.66111111,-5.833333333,2007,2009,Trough,50,7.5,510120,Wet,175,0.399543379,270,376.03035,413.2668244,8265.336487,270,404.5069595,8090.139191,Torresol,UTE Valle I,Sener,Sener,FIT,0.27,0.37603035,0.413266824,0.27,0.290446979,0.202978852,0.295587495,0.198676384,0.289322036,#N/A,Torresol,25,Lillestam et al 2017: Nature energy
+Helioenergy 2,2012,operating,Spain,Eclija,37.576247,-5.153659,2068,#N/A,Trough,50,0,300000,Wet,95,0.216894977,240,308.56728,332.9893758,6659.787515,240,325.7690809,6515.381618,Abengoa,Abener; Teyma,Schott,Abengoa,FIT,0.27,0.34713819,0.374613048,0.27,0.285051778,0.301276369,0.438733034,0.294743716,0.429219872,#N/A,Abengoa; Eon,25,Lillestam et al 2017: Nature energy
+Solaben 2,2012,operating,Spain,Logrosan,39.22472222,-5.390555556,2076,#N/A,Trough,50,0,300000,Wet,100,0.228310502,229,294.424613,317.7273627,6354.547254,229,310.837998,6216.759961,Abengoa,Abener; Teyma,Schott,Abengoa,FIT,0.27,0.34713819,0.374613048,0.27,0.285051778,0.273094476,0.397693215,0.267172898,0.38906993,#N/A,Abengoa (70%); Itochu (30%),25,Lillestam et al 2017: Nature energy
+Solaben 3,2012,operating,Spain,Logrosan,39.22472222,-5.390555556,2076,#N/A,Trough,50,0,300000,Wet,100,0.228310502,229,294.424613,317.7273627,6354.547254,229,310.837998,6216.759961,Abengoa,Abener; Teyma,Schott,Abengoa,FIT,0.27,0.34713819,0.374613048,0.27,0.285051778,0.273094476,0.397693215,0.267172898,0.38906993,#N/A,Abengoa (70%); Itochu (30%),25,Lillestam et al 2017: Nature energy
+Solacor 1,2012,operating,Spain,El Carpio,37.957172,-4.492179,2042,#N/A,Trough,50,0,300000,Wet,100,0.228310502,229,294.424613,317.7273627,6354.547254,229,310.837998,6216.759961,Abengoa,Abener; Teyma,Schott,Abengoa,FIT,0.27,0.34713819,0.374613048,0.27,0.285051778,0.273094476,0.397693215,0.267172898,0.38906993,#N/A,Abengoa (76%); JGC (24%(),25,Lillestam et al 2017: Nature energy
+Solacor 2,2012,operating,Spain,El Carpio,37.957172,-4.492179,2042,#N/A,Trough,50,0,300000,Wet,100,0.228310502,229,294.424613,317.7273627,6354.547254,229,310.837998,6216.759961,Abengoa,Abener; Teyma,Schott,Abengoa,FIT,0.27,0.34
```

Comment:
This is weird -- and actually it might let some tools fail reading the file (it does let my Atom plugin fail). You did a hard job removing the whitespace and quoting characters from the header, so we should not have them here.

```
"BrightSource Energy (25%) 
General Electric (25%) 
NOY Infrastructure & Energy Investment Fund (50%)
General Electric (25%) 
NOY Infrastructure & Energy Investment Fund (50%)"
```

should be

```
BrightSource Energy (25%) General Electric (25%) NOY Infrastructure & Energy Investment Fund (50%) General Electric (25%) NOY Infrastructure & Energy Investment Fund (50%)
```
.

(this actually let's me reconsider our quoting strategy... I'll write another comment for that